cmd/yggctl/actions.go

```diff
@@ -164,7 +167,7 @@ func messageJournalAction(ctx *cli.Context) error {
 				entry["worker_data"],
 			)
 		}
-		if err := writer.Flush(); err != nil {
+		if err = writer.Flush(); err != nil {
```

Comment:
Semantic nitpick.

```suggestion
		if err := writer.Flush(); err != nil {
```

I’d rather keep `:=`. It introduces a new temporary value for the `if` block. `=` assigns a new value to the existing `err` redeclared by the `fmt.Fprint` on the line after `tabwriter.NewWriter`.

Note that there, you use `:=` too. It’s also already used many times in the function above the `switch` statement.

The linter doesn’t complain about `:=`.